addons/calendar/static/tests/activity.test.js

```diff
@@ -1,12 +1,18 @@
-/* @odoo-module */
+import { test } from "@odoo/hoot";
+import { defineCalendarModels } from "@calendar/../tests/calendar_test_helpers";
+import {
+    click,
+    contains,
+    openFormView,
+    registerArchs,
+    start,
+    startServer,
+} from "@mail/../tests/mail_test_helpers";
 
-import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+defineCalendarModels();
+registerArchs({ "calendar.event,false,calendar": `<calendar date_start="start"/>` });
```

Comment:
`archs` are cleared with the `after` callback, it should be done in the test itself.